src/Morph/Web/MorphWebView.qml

```diff
@@ -48,6 +48,42 @@ WebEngineView {
         certificateError = null
     }
 
+    /**
+       html select override
+       set enableSelectOverride to true to make Morph.Web handle select
+       note that as it uses javascript prompt,
+       make sure that onJavaScriptDialogRequested signal handler don't overplay prompt dialog by checking the isASelectRequest(request)
+      **/
```

Comment:
Note the format of other documentation comments in this file:

* Each line begins with an asterisk
* There is a distinct character limit
* The comment ends with a single asterisk and a slash

Please replicate https://github.com/ubports/morph-browser/pull/273/files#diff-f88008971a9cf96598fe24671fe22d53R87, for example.